.github/workflows/cleanup-orphans.yml

```
@@ -0,0 +1,114 @@
+          cd apps/backend
+          echo "Fetching backend .env from 1Password..."
+          op read "op://${{ secrets.OP_VAULT_ID }}/${{ steps.env-config.outputs.env_item_id }}/notesPlain" > .env
+        env:
+          OP_SERVICE_ACCOUNT_TOKEN: ${{ secrets.OP_SERVICE_ACCOUNT_TOKEN }}
+
+      - name: Assess orphaned S3 files 🔬
+        run: npm run s3:assess-orphans
+
+  # This job requires a manual approval via the "cleanup-approval" environment.
+  # Go to: Settings > Environments > cleanup-approval > add required reviewers.
+  approve:
```

actionlint (1.7.11)

[error] "steps" section is missing in job "approve"

(syntax-check)